torchft/process_group_test.py

```diff
@@ -178,6 +193,17 @@ def test_baby_gloo(self) -> None:
 
         torch.testing.assert_close(at, bt)
 
+    def test_baby_gloo_timeout(self) -> None:
+        store = TCPStore(
+            host_name="localhost", port=0, is_master=True, wait_for_workers=False
+        )
+
+        store_addr = f"localhost:{store.port}/prefix"
+
+        a = ProcessGroupBabyGloo(timeout=timedelta(seconds=0.01))
+        with self.assertRaisesRegex(TimeoutError, "timed out after 0.01 seconds"):
+            a.configure(store_addr, 0, 2)
+
```

Comment:
Same suggestion as the gloo one about testing that timeout was changed

Reply:
since we have the time in the message I think that's sufficient